React/events.md

```diff
@@ -8,84 +8,112 @@
 React supplies support on many different events. We'll focus on a few of the common events, keyboard and click, but just know that you should reference the [full list](https://reactjs.org/docs/events.html#supported-events) if you want more information.
```

Comment:
Do you think it'd be worth adding screenshots to show how the website is changing as we make these changes? 
Our lessons aren't consistent about whether or not we do that, so I definitely don't think it's essential. It just occurred to me it's hard to follow what's happening in this lesson without seeing the visual changes. But maybe the expectation is that the student/teacher is following along with their own react project.

Reply:
I think the expectation is that the teacher is doing this along with the student, and it's hard to see the result because this lesson is based on events, but I can add a pair of'em.

Reply:
And done